jerry-core/ecma/operations/ecma-boolean-object.c

```diff
@@ -42,13 +44,27 @@ ecma_value_t
 ecma_op_create_boolean_object (ecma_value_t arg) /**< argument passed to the Boolean constructor */
```

Comment:
An important question: is this only used by constructors? Because if it is used elsewhere, it must not use the new target. This is also true for all other functions.

Reply:
Yes, only used `ecma_builtin_boolean_dispatch_construct`.